packages/docs/src/components/DocsComponent.vue

```diff
@@ -1,6 +1,6 @@
 <template>
```

Comment:
Perhaps we can also mark version for now? And also group items somehow?

![image](https://user-images.githubusercontent.com/5394573/180832258-120fddd8-d4cf-4bec-9508-394cfaa839ef.png)

Would be also nice to link issue, so that user can hover/click on card.